swagger_to_sdk_config.json

```diff
@@ -1,19 +1,66 @@
 {
-  "$schema": "https://raw.githubusercontent.com/lmazuel/swagger-to-sdk/master/swagger_to_sdk_config.schema.json",
-  "meta": {
-    "autorest_options": {
-      "version": "V2",
-      "csharp": "",
-      "reflect-api-versions": "",
-      "license-header": "MICROSOFT_MIT_NO_VERSION",
-      "sdkrel:csharp-sdks-folder": "./sdk",
-      "use": "@microsoft.azure/autorest.csharp@2.3.82"
-    },
     "advanced_options": {
       "create_sdk_pull_requests": true,
-      "sdk_generation_pull_request_base": "integration_branch",
-      "main_branch": "master"
+      "run_mode": "onePassMultipleReadmeMd" // or multiplePassSingleReadmeMd
```

Comment:
Wouldn't this be a generate option? 

Maybe name this: "generation_call_mode": "one-per-config" | "one-for-all-configs"

Reply:
Why do we even need modes? If the script want to call something per config it can do it internally?